specifications/api/swagger.yaml

```diff
@@ -1398,18 +1398,59 @@ components:
         passphrase: *lenientPassphrase
         address_index: *addressIndex
 
-    TransactionMetadatum: &TransactionMetadatum
+    TransactionMetadataValue: &TransactionMetadataValue
```

Comment:
We need to confirm this with @AlexIOHK . In theory, 1.20.0 supports both approaches for metadata, the one we currently use with direct JSON conversion, and this one that is more of a JSON description of the metadata. 

I'd have preferred going for the JSON description initially, but we had to get partners started with something and went for the direct JSON conversion which was ready. If we change that, it'll break existing integrations (although that is rather trivially fixed).

Reply:
OK I have changed the API to use the "no schema" mapping from cardano-api. I think it's the same as before. The PR for changing to the detailed schema is #2147.